runtime/gateway.go

```diff
@@ -248,8 +250,13 @@ func CreateGateway(
 		return nil, err
 	}
 
-	if err := gateway.setupTracer(config); err != nil {
-		return nil, err
+	if opts.Tracer != nil && opts.TracerCloser != nil {
```

Comment:
can you please make it config driven as done below for not found handler

Reply:
Thanks! Updated